packages/storage-plus/src/prefix.rs

```diff
@@ -69,11 +70,20 @@ where
     T: Serialize + DeserializeOwned,
 {
     pub fn new(top_name: &[u8], sub_names: &[&[u8]]) -> Self {
+        Prefix::new_de_fn(top_name, sub_names, deserialize_kv)
+    }
+
+    pub fn new_de_fn(
```

Comment:
I would use a different name, but I see this extension, and good to keep the default behavior the same.

Definitely need more time to review this

Reply:
Yes, I was struggling when naming it.

Maybe `new_custom_de`?
Another option would be to keep only `new`, and add a helper method `set_de_fn` or so. But I think it's more confusing / unclear.

Reply:
Take your time to review this. It's a more complex variant of the unique index case, but in the end it's the same mechanism: when creating the `Prefix`, a custom deserialization function is passed, which handles the indirection. I couldn't make it simpler.

Reply:
I meant, the related PR for the multi index.